tools/serverpod_cli/lib/src/logger/loggers/std_out_logger.dart

```diff
@@ -233,12 +245,34 @@ String _wrapText(String text, int columnWidth) {
   var textLines = text.split('\n');
   List<String> outLines = [];
   for (var line in textLines) {
-    outLines.add(line.wordWrap(width: columnWidth));
+    var replaceFirstChar = _hasLeadingTrimCharacter(line);
+    // wordWrap(...) uses trim as part of its implementation which removes all
+    // leading trimmable characters.
+    // In order to preserve them we temporarily replace the first char with a
+    // non trimmable character.
+    if (replaceFirstChar) {
+      line = '@${line.substring(1)}';
```

Comment:
Question: Is the first char guaranteed to be a white space? If not we would lose that information here.

Reply:
Yes, the check we do above trims the first character of the text. And if the result of the trim is empty then we know that the first char would disappear once we call the wordWrap method of superstring. Therefore we can safely replace the symbol.